.github/workflows/ci.yml

```diff
@@ -630,7 +629,6 @@ jobs:
         if: >-
           matrix.modules != 'plugin/trino-singlestore'
           && ! (contains(matrix.modules, 'trino-delta-lake') && contains(matrix.profile, 'cloud-tests'))
-          && ! (contains(matrix.modules, 'trino-delta-lake') && contains(matrix.profile, 'gcs-tests'))
```

Comment:
All the cloud-tests run in different regions. BTW the combined runtime is much shorter than runtime of `cloud-tests` + `gcs-tests` since `gcs-tests` had a single test class which limited concurrent tests. So chances of crossing 30m is even lower. I'll however keep an eye on CI times for next few days for builds on `master` to make sure we don't regress.